src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -7077,6 +7077,16 @@ public object VisitParenExpression(ParenExpressionAst parenExpressionAst)
         {
```

Comment:
@rjmholt, your last commit had 4 failures in `PowerShell-CI-windows`
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:164 char:49
+                 $result = testexe -returncode 0 && Test-TerminatingBa ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:177 char:49
+                 $result = testexe -returncode 1 || Test-TerminatingBa ...
+                                                 ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:190 char:54
+                 $result = Test-TerminatingBadCommand && testexe -retu ...
+                                                      ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:203 char:54
+                 $result = Test-TerminatingBadCommand || testexe -retu ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:215 char:39
+                 testexe -returncode 0 && throw 'Bad' &
+                                       ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:222 char:49
+                 $result = testexe -returncode 0 && Write-Error 'Bad'  ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:234 char:76
+ ... $result = Get-ChildItem 'doesnotexist' -ErrorAction Ignore || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:250 char:50
+             $result = Test-TerminatingBadCommand || Write-Output 'Hel ...
+                                                  ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:264 char:54
+                 $result = Test-TerminatingBadCommand || Write-Output  ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:164 char:49
+                 $result = testexe -returncode 0 && Test-TerminatingBa ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:177 char:49
+                 $result = testexe -returncode 1 || Test-TerminatingBa ...
+                                                 ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:190 char:54
+                 $result = Test-TerminatingBadCommand && testexe -retu ...
+                                                      ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:203 char:54
+                 $result = Test-TerminatingBadCommand || testexe -retu ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:215 char:39
+                 testexe -returncode 0 && throw 'Bad' &
+                                       ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:222 char:49
+                 $result = testexe -returncode 0 && Write-Error 'Bad'  ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:234 char:76
+ ... $result = Get-ChildItem 'doesnotexist' -ErrorAction Ignore || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:250 char:50
+             $result = Test-TerminatingBadCommand || Write-Output 'Hel ...
+                                                  ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:264 char:54
+                 $result = Test-TerminatingBadCommand || Write-Output  ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:164 char:49
+                 $result = testexe -returncode 0 && Test-TerminatingBa ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:177 char:49
+                 $result = testexe -returncode 1 || Test-TerminatingBa ...
+                                                 ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:190 char:54
+                 $result = Test-TerminatingBadCommand && testexe -retu ...
+                                                      ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:203 char:54
+                 $result = Test-TerminatingBadCommand || testexe -retu ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:215 char:39
+                 testexe -returncode 0 && throw 'Bad' &
+                                       ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:222 char:49
+                 $result = testexe -returncode 0 && Write-Error 'Bad'  ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:234 char:76
+ ... $result = Get-ChildItem 'doesnotexist' -ErrorAction Ignore || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:250 char:50
+             $result = Test-TerminatingBadCommand || Write-Output 'Hel ...
+                                                  ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:264 char:54
+                 $result = Test-TerminatingBadCommand || Write-Output  ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:164 char:49
+                 $result = testexe -returncode 0 && Test-TerminatingBa ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:177 char:49
+                 $result = testexe -returncode 1 || Test-TerminatingBa ...
+                                                 ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:190 char:54
+                 $result = Test-TerminatingBadCommand && testexe -retu ...
+                                                      ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:203 char:54
+                 $result = Test-TerminatingBadCommand || testexe -retu ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:215 char:39
+                 testexe -returncode 0 && throw 'Bad' &
+                                       ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:222 char:49
+                 $result = testexe -returncode 0 && Write-Error 'Bad'  ...
+                                                 ~~
The token '&&' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:234 char:76
+ ... $result = Get-ChildItem 'doesnotexist' -ErrorAction Ignore || testexe ...
+                                                                ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:250 char:50
+             $result = Test-TerminatingBadCommand || Write-Output 'Hel ...
+                                                  ~~
The token '||' is not a valid statement separator in this version.

At D:\a\1\s\test\powershell\Language\Operators\BashControlOperator.Tests.ps1:264 char:54
+                 $result = Test-TerminatingBadCommand || Write-Output  ...
+                                                      ~~
The token '||' is not a valid statement separator in this version.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```